www/src/theme.js

```diff
@@ -184,6 +184,7 @@ const sharedStyles = {
       marginTop: 25,
       marginLeft: -30,
       marginRight: -30,
+      marginBottom: 25,
```

Comment:
It looks better with margin between code blocks and the content below it.

### Prev

<img width="400" alt="screen shot 2017-10-05 at 12 20 47 am" src="https://user-images.githubusercontent.com/1315101/31188311-127e08f4-a967-11e7-989c-f023fc027dbe.png">

### Curr

<img width="400" alt="screen shot 2017-10-05 at 12 52 12 am" src="https://user-images.githubusercontent.com/1315101/31188434-758dfad0-a967-11e7-9ee3-d18bfe195105.png">